vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -250,13 +257,13 @@ private void loadExperimentInformation(Experiment experiment) {
   }
 
   private void loadTagInformation(Experiment experiment) {
-    tagLayout.removeAll();
+    tagCollection.removeAll();
     List<String> tags = new ArrayList<>();
     experiment.getSpecies().forEach(species -> tags.add(species.value()));
     experiment.getSpecimens().forEach(specimen -> tags.add(specimen.value()));
     experiment.getAnalytes().forEach(analyte -> tags.add(analyte.value()));
-    tags.forEach(tag -> tagLayout.add(new Tag(tag)));
-    tagLayout.getElement().setAttribute("Title", String.join(" ", tags));
+    tags.forEach(tag -> tagCollection.add(new Tag(tag)));
+    tagCollection.getElement().setAttribute("Title", String.join(" ", tags));
```

Comment:
```suggestion
    tags.stream().map(it -> new Tag(it)).forEach(tagCollection::add);
```
nit: separates the two steps of tag creation and tag attachment